src/packages/cli/src/__tests__/doctor.test.ts

```diff
@@ -1,74 +1,86 @@
-import tempy from 'tempy'
+import * as FSJet from 'fs-jetpack'
+import { FSJetpack } from 'fs-jetpack/types'
 import path from 'path'
+import tempy from 'tempy'
 import { Doctor } from '../Doctor'
-import copy from '@apexearth/copy'
-import assert from 'assert'
-import stripAnsi from 'strip-ansi'
 
-describe('doctor', () => {
-  it('doctor should succeed when schema and db match', async () => {
-    const tmpDir = tempy.directory()
-    await copy({
-      from: path.join(__dirname, 'fixtures/example-project/prisma'),
-      to: tmpDir,
-      recursive: true,
-    })
-    const cwd = process.cwd()
-    process.chdir(tmpDir)
+const ctx: {
```

Comment:
The concept of jest ctx can be turned into a composition API over time with a lot of code-reuse potential I think. tbd but we had some success with this approach in Nexus. CC @Weakky

Reply:
Interesting pattern! Happy to try that out